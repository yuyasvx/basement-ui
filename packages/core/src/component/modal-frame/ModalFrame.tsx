import type { FC, PropsWithChildren } from 'react';

export interface ModalFrameProps {
  show: boolean;
}

export const ModalFrame: FC<PropsWithChildren<ModalFrameProps>> = (props) => {
  return <div style={{ background: '#C0C0C0' }}>{props.children}</div>;
};
