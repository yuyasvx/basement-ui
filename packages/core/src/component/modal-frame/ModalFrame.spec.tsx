import { render } from '@testing-library/react';
import { describe, expect, test } from 'vitest';
import { ModalFrame } from './ModalFrame';

describe('Snapshot Test', () => {
  test('renders h1 text', () => {
    const { container } = render(<ModalFrame show />);
    expect(container.childNodes).toMatchSnapshot();
  });
});
