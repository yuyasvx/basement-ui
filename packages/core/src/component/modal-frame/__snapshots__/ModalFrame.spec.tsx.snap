// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshot Test > renders h1 text 1`] = `
NodeList [
  <div
    style="background: rgb(192, 192, 192);"
  />,
]
`;
